src/app/containers/Image/index.test.jsx

```diff
@@ -103,10 +103,9 @@ describe('Image', () => {
     it('should render an image with a sizes attribute', () => {
       render(<ImageContainer sizes="100vw" {...data} />);
 
-      const imgEl = screen.getByAltText(
```

Comment:
Wouldn't this get the image? Curious why it needed to be updated.

Reply:
It retrieved the `<img />` tag previously, this needed updated to grab the `<source />` tag instead as the `sizes` attribute is now placed on `<source />` rather than `<img />`.